rickchurch/constants.py

```diff
@@ -3,18 +3,21 @@
 from decouple import config
 
 
+DISCORD_BASE_URL: str = "https://discord.com/api"
```

Comment:
`: str` typehint isn't needed here, we do it in the other once, because `decouple.config` returns a Union of many types, making it unclear which type the resulting variable will have, but with literal string here, that's not the case and it is redundant.

Reply:
OK no problem